### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ihor-jpeg**
- [DEMO LINK](https://ihor-jpeg.github.io/layout_stop-watch/)

### CODE REVIEW

**Comment by dima-tkachenko** [comment]
Your code lookes wery well)

**Comment by ihor-jpeg** [comment]
> Your code lookes wery well)
Thank you!

File: src/styles/blocks/clock.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      $hand-second-height,
+      $hand-second-width,
+      $hand-second-color
+    );
+
+    animation: tik-tock $clock-second-duration steps(60, end) infinite 0.3s;
+  }
+
+  &__origin {
+    @include circle($origin-size, $origin-color);
```
Comment (VinnieJ-2k20) [inline src/styles/blocks/clock.scss:33]: Nice use of mixin 👍

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job and cool looking design 👍

**Pull request closed without merging**